services/oc-opencor-base/metadata/metadata.yml

```diff
@@ -0,0 +1,36 @@
+name: opencor
+key: simcore/services/comp/opencor
+type: computational
+integration-version: 1.0.0
+version: 1.0.0
+description: OpenCOR is a cross-platform modelling environment, which can be used to organise, edit, simulate and analyze CellML and SED-ML files
+contact: zhuang@itis.swiss
+authors:
+  - name: Katie Zhuang
+    email: zhuang@itis.swiss
+    affiliation: IT'IS Foundation
+inputs:
+  model_url:
+    displayOrder: 1
+    label: model url
+    description: the URL of a CellML / SED-ML file you would like to run
+    type: string
+    defaultValue: https://models.physiomeproject.org/workspace/5f5/rawfile/125f548ce204c1d815298d2c8c1d9b774d89e3a7/mcintyre_richardson_grill_model_2001.sedml
+  config_file:
+    displayOrder: 2
+    label: configuration file (JSON)
+    description: Used to configure the simulation (using the information in the simulation section, if present), customise the initial state of the model (using the information in the parameters section, if present).
+    type: data:application/json
+    defaultValue: config.json
+    fileToKeyMap:
+      config.json: config_file
+  
+outputs:
+  output_1:
+    displayOrder: 1
+    label: simulation output
+    description: JSON file containing outputs of the simulation
```

Comment:
Replace with "JSON file containing the output of the simulation"?

Reply:
changed